README.md

```diff
@@ -1,24 +1,126 @@
+
+[ActivAir](https://github.com/alia-peterson/ActivAir) solves this problem by tapping into data from the [IQAir AirVisual API](https://www.iqair.com/us/). Users can view data for their current location, add and remove other locations, get advice on whether it's safe to recreate outdoors given the AQI for that area, and click to view trail recommendations for each location on [AllTrails](https://www.alltrails.com/). The project specifications can be found [here](https://frontend.turing.io/projects/module-3/stretch.html).
+
+### Motivation & Goals
+* Learn a new "stretch" technology independently. After learning React last month, we chose to dive into Vue.js to deepen our understanding of frameworks. We had 9 days to self-teach Vue.js and create our app.
+* Create an MVP that solves a problem for users
+* Gain more experience using external APIs
+
+---
+
+## Technologies
+JavaScript, Vue, RESTful APIs, Cypress, Local Storage, HTML, CSS
+
+---
+
+## Deployment
+TO DO [Deployment Link]()
+
+**To run locally:**
+1. Clone down this repo
+2. `npm install`
+3. `npm run start`
+
+---
+
+## Features
+* [Display Current Location](#Display-Current-Location)
+* [Display Other Locations](#Display-Other-Locations)
+* [Activity Recommendations](#Activity-Recommendations)
+* [View Trails](#View-Trails)
+
+
+#### Display Current Location
+When the site loads, the you'll see a form with options to get info for you current location or to choose a location from a dropdown list of available cities and states. If you click "Nearest Location," a card will show up and persist on refresh with the latest data so you can stay up to date with the AQI and weather for where you are.
+<p align = "center">
+<img width="1440" alt="homepage" src="">
+</p>
+  <details>
+    <summary>Under the Hood</summary>
+    The current location card data comes from our <code>fetch</code> to the AirVisual API IP geolocation endpoint. Conditional rendering in Vue.js allowed us to include "Current Location" copy at the top of the card without the delete button. We used <code>unshift</code> to ensure that the current location card always shows up first in the render even as other location cards are added. Once a current location card has been added from the form, it is also added to <code>localStorage</code> and will show up on refresh with data from the latest timestamp.
+  </details>
+
+#### Display Other Locations
+To see what the AQI and weather looks like in other areas, you can choose supported cities and states from the dropdown menus in the form. Once a new location has been added, it's automatically saved so that you can come back to it later on. You can also delete a location if you no longer want to view it. Each card can be individually refreshed to view the latest weather and AQI data.
```

Comment:
I don't know that it seems necessary, but we could consider including that the manual refresh was a decision made due to the limitations of our API calls.

Reply:
I think I added this in the "Challenges" section below 🙂